api/nodes/serializers.py

```diff
@@ -166,6 +166,7 @@ class NodeSerializer(JSONAPISerializer):
     preprint = ser.BooleanField(read_only=True, source='is_preprint')
     fork = ser.BooleanField(read_only=True, source='is_fork')
     collection = ser.BooleanField(read_only=True, source='is_collection')
+    access_requests_enabled = ser.BooleanField(read_only=False, required=False)
```

Comment:
Drive-by: I think we need a relationship on the node for this, and that relationship should be able to give meta on how many requests currently exist with the related_counts query parameter (see https://osf.io/73dq8/wiki/home/ )